.github/workflows/automation-nightly-tests.yml

```diff
@@ -92,7 +92,7 @@ jobs:
           QA_AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
           QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
       - name: Upload test log
-        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        uses: actions/upload-artifact@c7d193f32edcb7bfad88892161225aeda64e9392 # v4.0.0
```

Comment:
@momentmaker is this version compatible for us? I saw you mentioned elsewhere that it may not be.

Reply:
we should keep it at `v3.1.2`. see here:
https://github.com/actions/upload-artifact#v4---whats-new

Reply:
Good catch. I'll revert that commit - thanks guys

Reply:
Also reverted download-artifact for the same reasons.